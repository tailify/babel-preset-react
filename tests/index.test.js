'use strict';

const babel = require('@babel/core');

test('should not contain invalid rules', () => {
  const result = babel.transform('<Foo bar="baz" />', {
    babelrc: false,
    presets: ['./lib/index.js'],
    sourceType: 'script',
  });

  expect(result).toBeTruthy();
  expect(result.code).toMatchSnapshot();
});
