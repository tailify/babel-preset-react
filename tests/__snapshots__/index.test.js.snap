// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not contain invalid rules 1`] = `
"var _jsxFileName = \\"\\";
React.createElement(Foo, {
  bar: \\"baz\\",
  __source: {
    fileName: _jsxFileName,
    lineNumber: 1
  },
  __self: this
});"
`;
